src/main/java/sc/fiji/bdvpg/scijava/command/bdv/ViewTransformatorCommand.java

```diff
@@ -78,6 +78,6 @@ public void run() {
         affineTransform3D.rotate(1, rotateAroundY);
         affineTransform3D.rotate(2, rotateAroundZ);
 
-        new ViewTransformator(bdvh, affineTransform3D).run();
+        new ViewerTransformChanger(bdvh, affineTransform3D, true, 0).run();
```

Comment:
Do you think it can be useful to have another constructor wiht true and 0 being the default values ?

Reply:
Yes, I will do this.

Reply:
But I think we should not put `concatenateTransform = true` as a default, because I was in fact surprised when I saw that it was concatenating. I would thus put no default there but require it in the constructor.
Ok?